### ISSUE TITLE
`ModuleNotFoundError: No module named 'tensorflow.contrib.linalg'`

### ISSUE THREAD

**Issue by matthew-mcateer** [tensorflow/probability#190]
When it comes to using Google Colab with python3, the following error comes up whenever I try to import tensorflow_probability. The colabs tha I've been running download TFP anew each time they're run, with he option of choosing either the latest TFP Stable or the latest TFP-nightly. As of around 3 days ago, both of these options run ino the same error. 

It's worth noting that slightly older versions of tensorflow_probability (like what I have installed on my local linux machine), run this code with no problem, wich leads me to conclude that this was the result of a very recent change to tensorlfow probability.

Here is the full stack trace of the error
```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-b49dcc6cff5d> in <module>()
     46         pass
     47 
---> 48 import tensorflow_probability as tfp
     49 tfd = tfp.distributions
     50 tfb = tfp.bijectors

/usr/local/lib/python3.6/dist-packages/tensorflow_probability/__init__.py in <module>()
     19 
     20 # from tensorflow_probability.google import staging  # DisableOnExport
---> 21 from tensorflow_probability.python import *  # pylint: disable=wildcard-import
     22 from tensorflow_probability.python.version import __version__

/usr/local/lib/python3.6/dist-packages/tensorflow_probability/python/__init__.py in <module>()
     20 
     21 from tensorflow_probability.python import bijectors
---> 22 from tensorflow_probability.python import distributions
     23 from tensorflow_probability.python import edward2
     24 from tensorflow_probability.python import glm

/usr/local/lib/python3.6/dist-packages/tensorflow_probability/python/distributions/__init__.py in <module>()
     75 from tensorflow_probability.python.distributions.sinh_arcsinh import SinhArcsinh
     76 from tensorflow_probability.python.distributions.truncated_normal import TruncatedNormal
---> 77 from tensorflow_probability.python.distributions.vector_diffeomixture import quadrature_scheme_softmaxnormal_gauss_hermite
     78 from tensorflow_probability.python.distributions.vector_diffeomixture import quadrature_scheme_softmaxnormal_quantiles
     79 from tensorflow_probability.python.distributions.vector_diffeomixture import VectorDiffeomixture

/usr/local/lib/python3.6/dist-packages/tensorflow_probability/python/distributions/vector_diffeomixture.py in <module>()
     26 from tensorflow_probability.python.distributions import seed_stream
     27 from tensorflow_probability.python.internal import distribution_util
---> 28 from tensorflow.contrib.linalg.python.ops import linear_operator_addition as linop_add_lib
     29 from tensorflow.python.framework import tensor_util
     30 from tensorflow.python.ops import control_flow_ops

ModuleNotFoundError: No module named 'tensorflow.contrib.linalg'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------
```

**Comment by joeyhaohao** [comment]
Hi Matthew, it seems that `tensorflow.contrib.linalg` has been moved to `tensorflow.python.ops.linalg` in the latest version of tensorflow (check this https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/ops/linalg). Upgrade tensorflow-probability to the latest version can help solve this issue.

**Comment by csuter** [comment]
Copying response from email here. Closing for now; if the tfp-nightly doesn't work for some reason, please reopen and we'll take another look.

I think the issue is that colab pins to the absolute newest TF version, including RC's. TF just released 1.12.0-rc0 a couple days ago. That probably made the contrib.linalg package disappear. I thought we had a patch to work around this in the 0.4 TFP release, but apparently not. I just created a new colab on colab.research.google.com and confirmed that I get the same error with `!pip install tensorflow-probability`, but if I restart the runtime and use `!pip install tfp-nightly` instead, the error does not occur.

**Comment by aviogit** [comment]
[This](https://github.com/tensorflow/magenta/issues/1350#issuecomment-439683048) worked for me.
